### ISSUE TITLE
[v3] "ProfileModifier.php does not comply with psr-4 autoloading standard"

### ISSUE THREAD

**Issue by cedx** [Intervention/image#1231]
I've upgraded to v3.0.0 today, and everything works. Good job ! Thanks a lot for this library.

The issue : when running "composer update", I get this warning...

```shell
`src\Drivers\Gd\Modifiers\ProfileModifier.php` does not comply with psr-4 autoloading standard. Skipping.
```

PHP 8.3.0 on Windows 11.
Composer 2.6.5.

**Comment by nolsen89** [comment]
Not sure if it will do the trick, but i found that 
vendor\intervention\image\src\Drivers\Gd\Modifiers\ProfileModifier.php
had a wrong namespace, try change it to this
namespace Intervention\Image\Drivers\Gd\Modifiers;

**Comment by olivervogel** [comment]
Namespace is fixed in 14db5290f2547fbc273d3ae709cd46ae84dbd0f1.
